CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+Thank you for your interest in contributing to Glosario!
+We welcome contributions of all kinds.
+If you're here to submit a definition,
+either for a new term
+or in another language for an existing term,
+please take a moment to read the guidance below.
+
+## Contribution Workflow
+
+When you're making your contribution(s) to Glosario, please keep the following
+guidance in mind:
+
+1. **Check the [Issues page of the Glosario repository][issues]** to see what terms and definitions other contributors are already working on. This will help you avoid accidentally spending time writing the same term(s)/definition(s) as someone else.
+2. When you know what term or definition you'd like to add, tell others about it by writing **a [new issue] on the main Glosario repository.**
```

Comment:
Please specify the language and terms in the title of the issue and/or pull request, e.g., `Romanian: logistic regression`.

Reply:
Having said that, I suspect people will find this onerous: if I want to add a single term, I'm not going to file the issue and then write the PR - I'm just going to write the PR. Can we ask them to put language/term in the subject of the PR and then also ask contributors to search those as well as issues before contributing? Issues can then be used if people are planning to write multiple definitions, e.g., "Romanian/relational database terms".

Reply:
The thinking behind this was to reduce redundancy of effort if many people want to contribute simultaneously (as could be the case during Hacktoberfest): it can be frustrating to spend time working on something only to find someone else was doing exactly the same thing. (This has happened to me before.) Although I hope this project attracts many contributions far into the future, I doubt such measures will be necessary permanently.